solutions/bronze/cf-1808B.mdx

```diff
@@ -54,4 +54,40 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+from collections import defaultdict
+
+for _ in range(int(input()))
+	n, m = map(int, input().split())
+	matrix = []
+	for _ in range(n):
+		matrix.append(list(map(int, input().split())))
+
+	# Grouping columns using a dictionary
+	hashmap = defaultdict(list)
+	for i in range(n):
+		for j in range(m):
+			hashmap[j].append(matrix[i][j])
+
+	res = 0
+	for i in range(m):
+		temp = hashmap[i]
+		temp = sorted(temp, reverse=True)
+		suf = []
+		prev = 0
+		for i in reversed(temp):
+			suf.append(prev)
+			prev += i
+
+		ans = 0
+		for i in range(n):
+			ans += temp[i] * (n - i - 1) - suf[n - i - 1]
+		res += ans
+
+	print(res)
```

Comment:
you have a variable called `ans`, so why are you printing `res`?

Reply:
Ans is for local sum and res is for global sum

Reply:
man that's kinda misleading
switch the two variables, and name `res` something more descriptive

Reply:
Ok i will update variables with descriptive names. Do i need to close this request or should i just use git pull and then push again

Reply:
the second one